frontend/www/js/omegaup/components/admin/Roles.vue

```diff
@@ -30,24 +30,37 @@
   </div>
 </template>
 
-<script>
-import {T} from '../../omegaup.js';
+<script lang="ts">
+import { Vue, Component, Prop, Emit } from 'vue-property-decorator';
+import { T } from '../../omegaup.js';
+import omegaup from '../../api.js';
+
+@Component({})
+export default class AdminRoles extends Vue {
+  @Prop() initialRoles!: omegaup.Role[];
+  @Prop() initialGroups!: omegaup.Group[];
+
+  T = T;
+  roles: omegaup.Role[] = this.initialRoles;
+  groups: omegaup.Group[] = this.initialGroups;
+
+  @Emit()
+  onChangeRole(ev: Event, role: string): omegaup.Role {
+    const selectedRole: omegaup.Role = {
```

Comment:
se enoja el compilador si haces

```suggestion
    return {
```

?

Reply:
Parece que no hay problema.